Tasks/SqlServerDacpacDeployment/task.json

```diff
@@ -0,0 +1,215 @@
+{
+    "id": "70A3A82D-4A3C-4A09-8D30-A793739DC94F",
+    "name": "SqlServerDacpacDeployment",
+    "friendlyName": "[Deprecated] SQL Server Database Deploy",
+    "description": "Deploy SQL Server Database using DACPAC",
+    "helpMarkDown": "[More Information](https://aka.ms/sqlserverdacpackreadme)",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 18
```

Comment:
Please confirm if the patch version was updated.

Reply:
yes, it was bumped up